test/CMakeLists.txt

```
@@ -70,12 +70,19 @@ set(stdexec_test_sources
     exec/async_scope/test_spawn_future.cpp
     exec/async_scope/test_empty.cpp
     exec/async_scope/test_stop.cpp
+    $<$<BOOL:STDEXEC_ENABLE_TBB>:tbbexec/test_tbb_thread_pool.cpp>
     )
 
 add_executable(test.stdexec ${stdexec_test_sources})
 
 target_include_directories(test.stdexec PRIVATE ${CMAKE_CURRENT_LIST_DIR})
-target_link_libraries(test.stdexec PUBLIC STDEXEC::stdexec stdexec_executable_flags Catch2::Catch2)
+target_link_libraries(test.stdexec
+    PUBLIC
+    STDEXEC::stdexec
+    $<$<BOOL:STDEXEC_ENABLE_TBB>:tbbexec>
+    $<$<BOOL:STDEXEC_ENABLE_TBB>:TBB::tbb>
```

This should be enough (`STDEXEC::` only for consistency with the `STDEXEC::stdexec` dependency above):
```suggestion
    $<$<BOOL:STDEXEC_ENABLE_TBB>:STDEXEC::tbbexec>
```